src/i18n/en/index.ts

```diff
@@ -0,0 +1,89 @@
+import type { BaseTranslation } from '../i18n-types';
+
+const en = {
+	allowConnections: 'Change your server settings to allow connections from',
+	allowMixedContent: 'Allow mixed content',
+	areYouSureYouWantToDeleteAll: 'Are you sure you want to delete all {type:string}?',
```

Comment:
You can add type definitions to the interpolated variables, which is pretty nifty!